packages/js/product-editor/src/style.scss

```diff
@@ -5,6 +5,6 @@
 @import 'components/details-categories-field/create-category-modal.scss';
 @import 'components/header/style.scss';
 @import 'components/block-editor/style.scss';
-@import 'components/section/style.scss';
-@import 'components/tab/style.scss';
+@import 'components/section/editor.scss';
+@import 'components/tab/editor.scss';
 @import 'components/tabs/style.scss';
```

Comment:
Could we actually remove the block specific styles now? Or is this done as a follow up?
Given we would have some duplicate styles if we are also loading in the main `style.scss` from the `product-editor` package.
It does look like the `block-library` package from Wordpress still includes them here, so maybe its fine?

Reply:
Currently the assets are registered, but not yet enqueued on the page.  This may prove useful for 3PDs, though I'm not entirely set on the route that we want to proceed with these assets.

We can continue to bundle them in `wc-product-editor-css` as we do now since we know we'll continue using all blocks and this will result in a single stylesheet which is probably more performant than each block having its own stylesheet.  But with caching, the differences may be negligible.

Let's discuss which path we'd like to use tomorrow in our call and we can address in a follow-up.

Reply:
Actually, I forgot the registration of styles occurs in [this PR](https://github.com/woocommerce/woocommerce/pull/37339), so this PR only handles building those assets are not used at all yet.

But my same thoughts above apply to that PR as well.

Reply:
> We can continue to bundle them in wc-product-editor-css as we do now since we know we'll continue using all blocks and this will result in a single stylesheet which is probably more performant than each block having its own stylesheet. But with caching, the differences may be negligible.

Yeah that would work, it may also be beneficial to expose another `css` file that exports all the none block styles, or just the editor related styles?